CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing
+
+## Writing
+
+Write your article in [kramdown](https://kramdown.gettalong.org/).
+Add your writeup to `_posts/`
+with the following front matter,
+and add a 500x500 cover art under `assets/images/`.
+
+```
+---
+layout:     post
+title:      on Lil Yachty - Minnesota
+author:     Ming Li
+date:       2017-10-08
+categories: lil yachty
+album_art:  assets/images/lil_boat.jpg
+---
+```
+
+## Guidelines
```

Comment:
Forgot to add this to my review, but going off of what Michael said, we should add a guideline to include some link to the music under review, either a hyperlink to the song on YouTube and/or a Apple Music/Spotify embed. If the review discusses the music video, embedding the video from YouTube is valid as well.

Reply:
Got it, I'll add a note